pkg/networkmanager/firewall-client.js

```diff
@@ -339,10 +339,14 @@ firewall.addServices = (zones, services) => {
 };
 
 firewall.removeServiceFromZones = (zones, service) => {
-    // We can't use Promise.all() here until cockpit is able to dispatch es2015 promises
-    // https://github.com/cockpit-project/cockpit/issues/10956
-    // eslint-disable-next-line cockpit/no-cockpit-all
-    return cockpit.all(zones.map(z => firewall.removeService(z, service)));
+    let lastPromise;
+    zones.map(z => {
+        if (lastPromise)
+            lastPromise = lastPromise.then(() => firewall.removeService(z, service));
```

Comment:
Clever :-)

What's the rationale for this, can you please mention this in the commit log?